synapse/handlers/federation_event.py

```diff
@@ -919,15 +915,13 @@ async def _get_state_after_missing_prev_event(
         desired_events = set(state_event_ids)
```

Comment:
there's a comment above this line "# start by just trying to fetch the events from the store" which isn't really right any more. Could you update it?